integration_tests/array_expr_01.py

```diff
@@ -0,0 +1,42 @@
+from ltypes import i32, f32, f64
+from numpy import empty, reshape
+
+def array_expr_01():
+    dim1: i32
+    dim2: i32
+    dim3: i32
+    dim1d: i32
+    i: i32
+    shape1d: i32[1] = empty(1, dtype=int)
+    shape3d: i32[3] = empty(3, dtype=int)
+    eps: f64
+    eps = 1e-12
+
+    dim1 = 10
+    dim2 = 10
+    dim3 = 5
+    dim1d = dim1 * dim2 * dim3
+
+    # TODO: Replace with dim1, dim2, dim3 in array type annotations
+    e: f64[10, 10, 5] = empty((dim1, dim2, dim3))
```

Comment:
This might be the default for `empty`, so not strictly needed, but should not hurt:
```suggestion
    e: f64[10, 10, 5] = empty((dim1, dim2, dim3), dtype=float64)
```